index.txt

```diff
@@ -22,15 +22,23 @@
 #
 #	$ curl https://getcaddy.com | bash -s git,mailout
 #
+# Or it can be run after downloading:
+#
+# $ wget getcaddy.com -O getcaddy.sh
+# $ bash getcaddy.sh git,mailout
```

Comment:
I'd like to keep these comments at the beginning as short as possible, and I think that people who are concerned about this already know to do this, so let's remove this.